website/docs/WhatIsWebdriverIO.md

```diff
@@ -3,7 +3,7 @@ id: what-is-webdriverio
 title: What is Webdriver.IO?
 ---
 
-WebdriverIO is a progressive automation framework built to automate modern web and mobile applications. It simplifies the interaction with your app and provides a set of plugins that help you create a scalable, robust and flakiness test suite.
+WebdriverIO is a progressive automation framework built to automate modern web and mobile applications. It simplifies the interaction with your app and provides a set of plugins that help you create a scalable, robust and stable test suite.
```

Comment:
`deterministic`/`solid` could be also a good one - an acceptable antonym for `flakiness`. Or maybe the two adjectives before in the sentence (`scalable, robust`) are enough?